app/providers/operations/conversation/delete/index.js

```diff
@@ -14,9 +14,19 @@ class ConversationDeleteOperation {
 
     const { conversation, participantIds } = await this.#hasAccess(conversationId, currentUserId)
 
-    await this.conversationService.removeParticipants(conversation, [currentUserId], participantIds)
+    const { removedIds: removedUserIds } = await this.conversationService.removeParticipants(
+      conversation,
+      [currentUserId],
+      participantIds
+    )
+
+    const removedUserIdsStrings = removedUserIds.map((id) => id.toString())
```

Comment:
At some point we have to think how to get rid of all these `toString()` statements